src/EMS/RingdownStatus.js

```diff
@@ -7,10 +7,17 @@ import Alert from '../Components/Alert';
 
 function RingdownStatus({ className, onStatusChange, ringdown }) {
   const [showUpdate, setShowUpdate] = useState(false);
```

Comment:
Let's rename this to showCancel, setShowCancel so it better matches the functionality implemented